publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -179,6 +179,13 @@ public Publication createPublicationFromImportedEntry(Publication inputData, Imp
         return insertResource(newResource);
     }
 
+    public Publication updatePublicationByImportEntry(Publication publication, ImportSource importSource) {
+        if (nonNull(importSource)) {
+            publication.addImportDetail(new ImportDetail(Instant.now(), importSource));
+        }
+        return updateResourceService.updatePublicationButDoNotChangeStatus(publication);
```

Comment:
Do we want the same timestamp as in modifiedDate?

Reply:
I guess that would be best, if it's not too much hassle